bridge_adaptivity/module/forms.py

```diff
@@ -30,7 +30,10 @@ class Meta:
         model = Activity
         exclude = ['collection', 'points']
         widgets = {
-            'stype': forms.HiddenInput(), 'points': forms.HiddenInput(), 'lti_lms_platform': forms.HiddenInput()
+            'stype': forms.HiddenInput(),
+            'points': forms.HiddenInput(),
+            'lti_lms_platform': forms.HiddenInput(),
+            'difficulty': forms.NumberInput(attrs={'step': "0.1", "min": 0.0, "max": 1.0})
```

Comment:
We can add a trailing comma. This will make a smaller diff for the following changes. But it's up to you.

Reply:
You are right. I fixed this.